features/admin.webhooks.v1/components/webhook-channel-config-form.scss

```
@@ -48,24 +49,60 @@
             margin: 0;
             display: flex;
             align-items: flex-start;
+            width: 100%;
 
             .MuiCheckbox-root {
                 padding: 0 !important;
                 margin: 0 !important;
                 margin-right: 8px !important;
+                flex-shrink: 0;
             }
         }
 
         .checkbox-label {
-            flex: 1;
-            margin-top: 0;
-            padding-top: 0;
-
-            .channel-name {
-                line-height: 1.2;
-                margin: 0;
-                padding: 0;
-                font-size: 14px;
+            width: 100%;
+
+            .channel-name-container {
+                display: flex;
+                align-items: flex-start;
+                gap: 8px;
+                margin-bottom: 4px;
+                width: 100%;
+                min-height: 1.2em;
+
+                .channel-name {
+                    line-height: 1.2;
+                    margin: 0;
+                    padding: 0;
+                    font-size: 14px;
+                    flex: 1;
+                    min-width: 0;
+                    word-wrap: break-word;
+                    overflow-wrap: break-word;
+                    hyphens: auto;
+                }
+
+                .channel-status-inline {
+                    display: flex;
+                    align-items: center;
+                    gap: 4px;
+                    font-style: italic;
+                    font-size: 0.875rem;
+                    flex-shrink: 0;
+                    white-space: nowrap;
+
+                    &.success {
+                        color: #4caf50;
```

Lets try to replace this with the theme's success color.

```suggestion
                        color: "var(--oxygen-palette-success-main)";
```